packages/mongodb-chatbot-server/src/routes/responses/createResponse.ts

```diff
@@ -185,7 +203,28 @@ export function makeCreateResponseRoute({
       if (max_output_tokens > maxOutputTokens) {
         throw makeBadRequestError({
           error: new Error(
-            MAX_OUTPUT_TOKENS_ERR_MSG(max_output_tokens, maxOutputTokens)
+            ERR_MSG.MAX_OUTPUT_TOKENS(max_output_tokens, maxOutputTokens)
+          ),
+          headers,
+        });
+      }
+
+      // --- LOAD CONVERSATION ---
+      const conversation = await loadConversationByMessageId({
+        messageId: previous_response_id,
+        conversations,
+        headers,
+      });
+
+      // TODO: abstract this logic and share between this and addMessageToConversation
```

Comment:
can u add a ticket for sharing logic between addMessageToConversation and createResponse? could also be useful to add ticket name in the comment so it's not lost

Reply:
Just going to fix actually!